client/src/shared/hooks/useSignOut.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { alertError, alertSuccess, getUserToken } from '@/src/shared/utils';
+import { useRouter } from 'next/router';
+import axios from 'axios';
+
+export const useSignOut = (): any => {
+  const router = useRouter();
+  const onSignOutEvent = async () => {
+    try {
+      const token = getUserToken();
+      await axios.post(
+        `${process.env.NEXT_PUBLIC_API_BASE_URL}auth/sign-out`,
+        null,
+        {
+          headers: {
+            Authorization: `Token ${token}`
+          }
+        }
+      );
+      // Remove token from localStorage
```

Comment:
- remove comment
- also remove from local storage the ff:
-- user_full_name
-- user_username
-- user_email